### DISCUSSION TITLE
Velero v1.6.0 has been released!

### DISCUSSION THREAD

**Discussion by zubron** [vmware-tanzu/velero#3675] [Announcements]
Hi Velero community :wave:

Today, we released Velero v1.6.0! :tada: 

We're really excited to bring the following features and improvements in this release:
 * Support for per-BSL credentials
 * Progress reporting for restores
 * Restore API Groups by priority level
 * Restic v0.12.0 upgrade
 * End-to-end testing 
 * CLI usability improvements

Please check the [release notes](https://github.com/vmware-tanzu/velero/releases/tag/v1.6.0) for more details and see [our documentation](https://velero.io/docs/v1.6/upgrade-to-1.6/) for details on upgrading to this version.

**Comment by carlisia** [comment]
Read the blog post for the highlights of this release: https://velero.io/blog/velero-1.6-bring-all-your-credentials. 🎉
